vertx-mongo-service/src/main/java/io/vertx/ext/mongo/impl/MongoServiceImpl.java

```diff
@@ -165,13 +159,6 @@ public MongoService findWithOptions(String collection, JsonObject query, FindOpt
     return this;
   }
 
-  @Override
-  public MongoService findBatchWithOptions(String collection, JsonObject query, FindOptions options,
-      Handler<AsyncResult<JsonObject>> resultHandler) {
-    client.findBatchWithOptions(collection, query, options, resultHandler);
-    return this;
-  }
-
   @Override
   @Fluent
```

Comment:
Can we remove the @Fluent in this class. It should not be used.

Reply:
What do you mean?

Reply:
@Fluent on a method in an implementation class is needed.